tests/unit/mapper.js

```diff
@@ -1,8 +1,10 @@
 'use strict'
 
+const rewire = require('rewire')
 const tap = require('tap')
 
-const {createMappedObject} = require('../../src/middleware/mapper')
+const mapper = rewire('../../src/middleware/mapper')
+const createMappedObject = mapper.__get__('createMappedObject')
 
 tap.test('Mapper', {autoend: true}, t => {
   t.test('createMappedObject()', {autoend: true}, t => {
```

Comment:
@bradsawadye Please could you replace this test with something to test that if the `inputMapping` field is null that the mapping doesn't fail?

Unless, @BMartinos this should fail if there's no mapping? The reason I changed this was because if the inputValidation or the external Requests objects were null we just logged a warn and continued.

Reply:
If there isnt any inputMapping supplied it shouldnt throw an error and rather just proceed to the next middleware. Mapping can be seen as optional and shouldnt be enforced